lib/osf-components/addon/components/delete-button/template.hbs

```diff
@@ -29,6 +29,8 @@
     <Button
         data-analytics-name='Delete button'
         data-test-delete-button
+        @layout={{this.buttonLayout}}
+        class={{this.class}}
         @type='destroy'
         {{on 'click' this._show}}
         @disabled={{this.disabled}}
```

Comment:
If you add `...attributes` here, you can use a `local-class` for `preprint-delete-btn`. While you're here, it may be a good idea to add `...attributes` after line 9 and line 21 as well.